app/controllers/advice_controller.rb

```diff
@@ -1,43 +1,60 @@
 class AdviceController < ApplicationController
```

Comment:
Can you add (as a code comment) what advice controller actually does in the overall application. You can basically copy and paste the content from your wiki page